src/backend/shortcuts/shortcuts/shortcuts.ts

```diff
@@ -52,7 +51,7 @@ async function addShortcuts(gameInfo: GameInfo, fromMenu?: boolean) {
     case 'linux': {
       const icon = await getIcon(gameInfo.app_name, gameInfo)
       const shortcut = `[Desktop Entry]
-Name=${removeSpecialcharacters(gameInfo.title)}
```

Comment:
Name allows all types of characters, removeSpecialcharacters felt unnecessary here